<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupWindow</class>
 <widget class="QWidget" name="StartupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GFXReconstruct Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>300</height>
    </rect>
   </property>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(111, 157, 236);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RecordButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(111, 157, 236);
color: rgb(244, 205, 249);</string>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReplayButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(111, 157, 236);
color: rgb(244, 205, 249);</string>
    </property>
    <property name="text">
     <string>Replay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(111, 157, 236);
color: rgb(244, 205, 249);</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>275</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(111, 157, 236);
color: rgb(244, 205, 249);</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>275</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(111, 157, 236);
color: rgb(244, 205, 249);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QListView" name="SelectListView">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(111, 157, 236);
color: rgb(244, 205, 249);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputLineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>246</y>
      <width>256</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(111, 157, 236);
color: rgb(244, 205, 249);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FileSelectButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>246</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(111, 157, 236);
color: rgb(244, 205, 249);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RemoveUnsupportedBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(244, 205, 249);</string>
    </property>
    <property name="text">
     <string>Remove Unsupported</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>CloseButton</zorder>
   <zorder>RecordButton</zorder>
   <zorder>ReplayButton</zorder>
   <zorder>OpenButton</zorder>
   <zorder>NextButton</zorder>
   <zorder>BackButton</zorder>
   <zorder>SelectListView</zorder>
   <zorder>InputLineEdit</zorder>
   <zorder>FileSelectButton</zorder>
   <zorder>RemoveUnsupportedBox</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
